src/extensions/auto-refresh/bootstrap-table-auto-refresh.js

```diff
@@ -8,6 +8,7 @@ const Utils = $.fn.bootstrapTable.utils
 
 $.extend($.fn.bootstrapTable.defaults, {
   autoRefresh: false,
+  showAutoRefresh: true,
```

Comment:
We need more logic for this option, right?

Reply:
No it's already implemented because of our [buttons](https://bootstrap-table.com/docs/api/table-options/#buttons) feature.
At all we don't really need that assignment there (because the default value of `show<name>` is always true), but i guess it's better for our understanding if we read the source code :-)